examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
<p>You can remove the TODO: just the protocol calls a PRG many times :) </p><p> </p><p>Mention that this table is extracted from the AriaNN paper maybe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Done. </p>